packages/hint-no-disallowed-headers/src/hint.ts

```diff
@@ -106,7 +106,6 @@ export default class NoDisallowedHeadersHint implements IHint {
             host: 'disallowedHostHeader',
             p3p: 'disallowedP3PHeader',
             pragma: 'disallowedPragmaHeader',
-            via: 'disallowedViaHeader',
```

Comment:
We should also clean up the string from the corresponding `messages.json`: https://github.com/webhintio/hint/blob/5ff878455533a5f44b0652154f39760c1b81be4d/packages/hint-no-disallowed-headers/src/_locales/en/messages.json#L30